source/Mission.cpp

```diff
@@ -1284,7 +1284,7 @@ Mission Mission::Instantiate(const PlayerInfo &player, const shared_ptr<Ship> &b
 		if(commodity)
 			result.cargo = commodity->items[Random::Int(commodity->items.size())];
 		else
-			result.cargo = cargo;
+			result.cargo = Phrase::ExpandPhrases(cargo);
```

Comment:
What if the phrases were expanded *before* considering if this is referring to a commodity type or not?

Reply:
Only if the phrase results in a match to one of the commodities.
And, if it does, maybe that's what the user wanted?

It might be better to do this part in another PR, though.
Commodities as they are used here are already essentially phrases, just very simple ones.
If we're going to support actual phrases here, we ought to deprecate the commodities because it's just a less flexible way of doing the same thing.

Reply:
I can't think of any way it would be ambiguous, so I implemented it.